packages/cloudflare/env.d.ts

```diff
@@ -5,6 +5,7 @@ declare global {
       __NEXT_PRIVATE_STANDALONE_CONFIG?: string;
       SKIP_NEXT_APP_BUILD?: string;
       NEXT_PRIVATE_DEBUG_CACHE?: string;
+      __OPENNEXT_KV_BINDING_NAME: string;
```

Comment:
@james-elicx Is this documented already? 
If not, would you mind sending a PR?
Also maybe add a default value.
Thanks!

Reply:
The intention was for this to just be used internally to inline a value into the cache handler, but yeah, it would make sense to let users use it as well